script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -15,7 +17,7 @@
 AppId={{286391DE-F778-44EA-9375-1B21AAA04FF0}
 AppName={#MyAppName}
 AppVersion={#MyAppVersion}
-;AppVerName={#MyAppName} {#MyAppVersion}
+AppCopyright=GitHub, Inc. and Git LFS contributors
```

Comment:
I did already verify this. You can also easily do it yourself, it you want, by simply downloading the [installer artifact from the AppVeyor build](https://ci.appveyor.com/api/buildjobs/ycarh44lni2qr7so/artifacts/git-lfs-windows-1.5.0.exe) (this does not have the most recent version number as in `master` we still are at `1.5.0`). See this screenshot:

![2016-12-13_084818](https://cloud.githubusercontent.com/assets/349154/21131797/f29da696-c110-11e6-86cf-dcb16437f885.jpg)